src/frequenz/dispatch/_dispatcher.py

```diff
@@ -178,11 +186,53 @@ def __init__(
             microgrid_id,
             self._client,
         )
+        self._actor_dispatchers: dict[str, ActorDispatcher] = {}
 
-    async def start(self) -> None:
+    def start(self) -> None:
         """Start the local dispatch service."""
         self._bg_service.start()
 
+    async def manage(
```

Comment:
I would call this `start_dispatching()` to make it more explicit and also add `stop_dispatching()` as we will need that when actors are disabled and should not be dispatched anymore (this is already done in the edge app).